public/index.html

```diff
@@ -3,17 +3,57 @@
 <head>
   <title>Word Scramble</title>
   <script src="bower_components/angular/angular.min.js"></script>
+  <link rel="stylesheet" href="appStyle.css">
 </head>
-<body>
-  <div class="main-container">
-    <h1>Word Scramble</h1>
-    <div class="app" ng-app="scrambleApp">
-    Hello
-      <div ng-controller="hihi">Yellow</div>
-    </div>
-
-    <script src="scripts/app.js"></script>
-    <script src="scripts/controllers/input.js"></script>
+<body ng-app="scrambleApp">
+  <div class="container" ng-app="scrambleApp">
+    <header class="header">
+      <h1 class="header-title">Word Scramble</h1>
+    </header>
+    <main class="game-container" ng-controller="mainController">
+
+      <div scramble-input class="start-screen text-screen" ng-if="state==='start'" ng-controller="startController">
```

Comment:
the front-end should be agnostic of the 'state' variable, rather you could do... ng-if="isStart()"